user-guide/Cloud_Platform/AboutCloudPlatform/About_the_Cloud_Platform.md

```diff
@@ -22,9 +24,39 @@ On the dataminer.services home page, you will find links to the different availa
 
 - [Admin](xref:CloudAdminApp)
 
+When your DMS has been connected to dataminer.services, you can also make use of the [DataMiner Teams bot](xref:DataMiner_Teams_bot).
+
+## System overview
+
 The home page also lists the DataMiner Systems that have been [connected to dataminer.services](xref:Connecting_your_DataMiner_System_to_the_cloud), provides links to different URLs for [remote access](xref:Cloud_Remote_Access), and allows you to [link your dataminer.services and DataMiner account](xref:Linking_your_DataMiner_and_DCP_account).
 
-When your DMS has been connected to dataminer.services, you can also make use of the [DataMiner Teams bot](xref:DataMiner_Teams_bot).
+### Connection states
+
+On the dataminer.services page you can see the state of the connection between your system and the dataminer.services platform. Next to the name of your DMS you will have an icon that will change color based on the state.
+
+#### DMS has a connection to dataminer.services
+
+Everything is okay and you can access your system and use all the features that the dataminer.services platform offers. This is indicated by the green colored icon icon and the green bar next to the DMS information.
+
+![Connection status ok](~/user-guide/images/DMS_status_overview_ok.png)
+
+#### DMS has a connection to dataminer.services but not online
+
+Your DMS has a connection to dataminer.services but the DMS itself is not running. This is indicated by the orange colored icon and the orange color next to the DMS information. When hovering over the cloud icon you will see the following tooltip: "Cloud connection valid, DMS not responding". You also get an indication of the last point in time where the system was detected as being online.
```

Comment:
Don't think we need to add this, as we can update the title at some point and will probably forget to update these docs for such a 'small' change